cpu/nrf52/include/openwsn_defs.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu
+ * @{
+ *
+ * @file
+ * @brief       nrf52 specific definitions for pkg/openwsn
+ *
+ * @author      Francisco Molina <francois-xavier.molina@inria.fr>
+ */
+
+#ifndef OPENWSN_DEFS_H
+#define OPENWSN_DEFS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifdef CPU_MODEL_NRF52840XXAA
```

Comment:
Why is this specific to nRF52840 ?

Reply:
The other models don't support ieee802.15.4 with the radio

Reply:
That's not true.
nRF52811, nRF52820, nRF52833 also have the 802.15.4 radio.

Reply:
I have not tested on the other radios, the timing could be different.

Reply:
They all use the same CPU frequency though

Reply:
No idea, if you think they would be the same I can add them, I had chosen to err on the safe side.. but I don't mind adapting

Reply:
I mean the alternative would be fall-back to the OpenWSN defaults - wouldn't that be worse?

Reply:
The defaults err on the safe side, so it should be fine.